Backend/src/API/REST/routes/user/userRouter.ts

```diff
@@ -50,13 +50,14 @@ const get_user_by_id = (): Router => {
   router.get("/users/:id", check_auth(), async (req: ExtRequest, res: any) => {
     try {
       const user_id = req.params.id;
-      const user = await UserService.getUserById(user_id);
+      const {email, password, isEmailVerified, isAdministrator,
+        ...filtered_user} = await UserService.getUserById(user_id);
```

Comment:
```suggestion
        ...filtered_user} = await UserService.getUserById(user_id) || {};
```
Destrucutring of user will crash if user is null/undefined